Gopkg.lock

```diff
@@ -31,12 +37,24 @@
   packages = ["pkg/ns"]
   revision = "7f98c94613021d8b57acfa1a2f0c8d0f6fd7ae5a"
 
+[[projects]]
+  name = "github.com/containers/virtcontainers"
```

Comment:
I guess we can just remove this now that virtcontainers lives in kata-containers/runtime/?

Reply:
This is auto-generated by `dep ensure`, Somehow is still resolving to the old repo as dependency. I'll check why.

Reply:
I think it's due to the `cc-runtime` sub-module.